bin/run.sh

```diff
@@ -1,26 +1,62 @@
-#! /bin/sh
-set -e
+#!/usr/bin/env bash
 
-SLUG="$1"
-INPUT_DIR="$2"
-OUTPUT_DIR="$3"
+# Synopsis:
+# Run the test runner on a solution.
 
-echo "$SLUG: testing..."
-# PLACEHOLDER - Insert call to run your language tests here
+# Arguments:
+# $1: exercise slug
+# $2: absolute path to solution folder
```

Comment:
The trailing slash part is probably still important. `absolute` only means it starts with a drive, mount or `/`